azure-pipelines.yml

```diff
@@ -31,10 +31,50 @@ variables:
     value: "ukhydrographicoffice/terraform-azure-powershell-unzip:1.1.4"
   - name: DeploymentPool
     value: "UKHO Ubuntu 1804"
+  - name: coverityPool
+    value: NautilusBuild
+  - group: Covscan-vars
+
+resources:
+ repositories:
+  - repository: covscan
+    type: github
+    name: UKHO/coverityscan-buildtemplates
+    endpoint: UKHO
+    ref: refs/heads/master
 
 stages:
+  - stage: CoverityScan
+    displayName: "Coverity Scan"
+    pool:
+      name: $(coverityPool)
+    jobs:
+    - job: Coverity
+      workspace:
+        clean: all
+      steps:
+        - checkout: self
+          submodules: recursive
+        - task: UseDotNet@2
+          displayName: 'Use .NET Core sdk'
+          inputs:
+            packageType: sdk
+            version: 3.1.x
```

Comment:
I notice global.json is specifying SDK v6. I have no idea if using an earlier SDK for Coverity will result in inconsistent issue reporting but perhaps it might be better to use SDK v6 here too?

Reply:
Updated to use .NET 6.0.x SDK